podpac/core/data/array_source.py

```diff
@@ -43,30 +45,23 @@ class Array(DataSource):
     >>> coords = podpac.Coordinates([podpac.clinspace(1, 10, 10, 'time'), 
                                      podpac.clinspace(1, 32, 32, 'lat'),
                                      podpac.clinspace(1, 34, 34, 'lon')])
-    >>> node = podpac.data.Array(source=data, native_coordinates=coords, outputs=['R', 'G', 'B'])
+    >>> node = podpac.data.Array(data=data, native_coordinates=coords, outputs=['R', 'G', 'B'])
     >>> output = node.eval(coords)
     """
 
-    source = ArrayTrait().tag(readonly=True)
-    native_coordinates = tl.Instance(Coordinates, allow_none=False).tag(attr=True)
+    data = ArrayTrait().tag(attr=True)
```

Comment:
I personally liked `source` because we use that everywhere. I agree `data` makes more sense here as `source` usually describes a `Node` or a `str`, but I just know it's going to trip me up.

Reply:
It think it's weird to have different child classes with different types for the "same" trait. Or, to use the same trait name for different traits. But I see where you're coming from.